spec/factories/contributors.rb

```diff
@@ -58,7 +58,7 @@
       after(:build) do |contributor|
         contributor.email = nil
         # FIXME: Faker seems to create invalid phone numbers sometimes
-        contributor.whats_app_phone_number = Faker::PhoneNumber.cell_phone
+        contributor.whats_app_phone_number = Faker::PhoneNumber.unique.cell_phone_in_e164
```

Comment:
These changes are attempting to address the issue with so many invalid phone numbers being created. it does not completely solve it. See comments in PR conversation.